corehq/apps/hqadmin/reports.py

```diff
@@ -1287,3 +1290,46 @@ def get_data(domains):
                 row[version_index + 1] = data.doc_count
 
         return rows.values()
+
+
+class AdminPhoneNumberReport(PhoneNumberReport):
+    name = ugettext_lazy("Admin Phone Number Report")
+    slug = 'phone_number_report'
+    fields = [
+        RequiredPhoneNumberFilter,
+    ]
+
+    dispatcher = AdminReportDispatcher
```

Comment:
Didn't work with inheriting `PhoneNumberReport` and `AdminReport` and multi-inheritance gets fairly confusing anyways. 

These are the only 3 attributes we would gain from inheriting `AdminReport`